### PR TITLE
Get all processes and stop task from build instance

### PR DESCRIPTION
**Pull request by Twsouza**
You cannot currently kill a stuck build process. `ProccessStop` only tries to kill processes in the `Instances` cluster, it should also try to kill the process in the `BuildInstances`.

### CODE REVIEW

File: provider/aws/processes_test.go [hunk 1]
```diff
@@ -264,8 +264,12 @@ func TestProcessRunDetached(t *testing.T) {
 
 func TestProcessStop(t *testing.T) {
 	provider := StubAwsProvider(
-		cycleProcessListTasksRunning,
```
Comment (heronrs) [inline provider/aws/processes_test.go:267]: why did you remove these?

File: provider/aws/processes_test.go [hunk 2]
```diff
@@ -1419,26 +1423,6 @@ var cycleProcessStopTask = awsutil.Cycle{
 	},
 }
 
-var cycleProcessStopTaskReason = awsutil.Cycle{
```
Comment (heronrs) [inline provider/aws/processes_test.go:1422]: i guess this is not used?

**Review by heronrs — APPROVED** [review]
lgtm, just have some questions

File: provider/aws/processes_test.go (hunk 2 above)
Comment (Twsouza) [reply provider/aws/processes_test.go:1422]: Yes, this is no longer used.

File: provider/aws/processes_test.go (hunk 1 above)
Comment (Twsouza) [reply provider/aws/processes_test.go:267]: Because I changed the method to get the process, before it only gets running and stopped, it didn't get the process from the BuildInstance.
The `cycleProcessListTasksRunning` and `cycleProcessListTasksStopped` don't satisfy the appTaskARNs request, I had to change the stub processes.

**Review by heronrs — APPROVED** [review]
LGTM
